### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by enes4949**
Not running npm install -g node-gyp as root/admin gives permission error.

##### Checklist

- [ ] `npm install && npm test` passes
- [ ] tests are included 
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)

### CODE REVIEW

**Review by rvagg — CHANGES_REQUESTED** [review]
You need a space before `(` and this isn't quite correct - many people (myself included) will set up their system so that their global node_modules is owned by their regular dev user account so `sudo` never needs to get involved. I'm not comfortable recommending root or `sudo` usage, although a suggestion about permissions and the need to overcome these problems might be welcome.

Something like:

> If your user account doesn't have access to the global `node_modules` (recommended) then this may need to be run as the root user or with `sudo`.

File: README.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ etc.), regardless of what version of Node.js is actually installed on your syste
 
 ## Installation
 
-You can install `node-gyp` using `npm`:
+You can install `node-gyp` using `npm`(run as root/admin):
 
 ``` bash
-$ npm install -g node-gyp
```
Comment (rvagg) [inline README.md:27]: also this change should be reverted or you're going to need to do it for all of the bash blocks in the README (which I'm not opposed to)

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Install the current version of Python from the [Microsoft Store package](https:/
 
 #### Option 1
 
-Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global --production windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator).
+Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install --global windows-build-tools` from an elevated PowerShell or CMD.exe (run as Administrator).
```
Comment (cclauss) [inline README.md:52]: Does windows-build-tools still install legacy Python?  If so then it should be removed in favor of installing Python 3 from the Microsoft App Store.

**Comment by cclauss** [comment]
Closing.  Out of sync with the current README.md.

**Pull request closed without merging**